import-dashboards.py

```diff
@@ -105,7 +148,7 @@ def add_datasources():
             sys.exit(-1)
 
     if 'CloudWatch' not in ds:
-        data = json.dumps({'name': 'CloudWatch', 'type': 'cloudwatch', 'jsonData': '{"defaultRegion":"us-east-1"}', 'access': 'proxy', 'isDefault': False})
```

Comment:
The root cause is there: `jsonData` should be a nested JSON object, not encoded JSON string. Grafana API does not validate it and stores as-is, leading to JS error on the edit page.